test/unit/test_domain_decomposition.cpp

```diff
@@ -444,7 +474,7 @@ TEST(domain_decomposition, unidirectional_gj_recipe) {
                 {},
                 {},
                 {},
-                {gap_junction_connection({4, "gj"}, {"gj"}, 0.1)},
+                {gap_junction_connection({4, "gj"}, {"gj"}, 0.1), gap_junction_connection({4, "gj"}, {"gj"}, 0.1)},
```

Comment:
why do we need this additional connection?

Reply:
It checks that duplicates are handled without hiccups.